hack/go-vet.sh

```diff
@@ -4,4 +4,4 @@ if [[ -z ${CI} ]]; then
     source hack/go-dep.sh
     operator-sdk generate k8s
 fi
-go vet ./...
\ No newline at end of file
+go vet ../...
```

Comment:
this doesn't look right... should it be `go vet ./...`?

Reply:
isn't it relative to the `hack` folder?

Reply:
ok, i haven't tested... checking.

Reply:
based on my testing, `go vet ./...` works as expected. that logic would also w/ the fact that you are issuing a relative `source hack/go-dep.sh` just above this.